dev/conductor/core/lib/src/state.dart

```diff
@@ -149,6 +149,9 @@ String phaseInstructions(pb.ConductorState state) {
         return <String>[
           'There are no engine cherrypicks, so issue `conductor next` to continue',
           'to the next step.',
+          '\n',
+          'Note: Create a new entry for this release in go/release-eng-retros',
```

Comment:
http://go/release-eng-retros will make it clickable in most terminals (making it easier to get to opening the doc)

Reply:
Thanks for the heads up on this.